spec/actions/mutation_spec.rb

```diff
@@ -1,3 +1,4 @@
+# rubocop:disable Metrics/BlockLength
 # frozen_string_literal: true
 
 RSpec.describe 'actions/mutation', :vcr do # rubocop:disable Metrics/BlockLength
```

Comment:
do you still need this `# rubocop:disable Metrics/BlockLength`, seems like its a duplicate now that you have disabled it for the whole file. Please check this.